core/control-plane/transfer-core/src/main/java/org/eclipse/edc/connector/transfer/process/TransferProcessManagerImpl.java

```diff
@@ -184,7 +184,7 @@ public StatusResult<TransferProcess> initiateConsumerRequest(TransferRequest tra
                 .dataRequest(dataRequest)
                 .type(CONSUMER)
                 .clock(clock)
-                .properties(dataRequest.getProperties())
+                .properties(transferRequest.getPrivateProperties())
```

Comment:
why don't have a `privateProperties` field on `TransferProcess`? I'm thinking this in the process of getting rid of `DataRequest` (in fact `DataRequest` properties are `TransferProcess` properties)

Reply:
I think we wanted to get rid of `DataRequest` as a whole, because at the end it's just an unneeded type, so the `TransferProcess` `properties` are the `DataRequest` properties (as it is currently), 
while the private properties should be a completely new field.

TL;DR: `TransferProcess` should have a `properties` field and a `privateProperties` one.

maybe something for a subsequent PR, just wanted to check if we're aligned on this

Reply:
:+1:  in this case in this PR i would just rename `TransferProcess#properties` to `TransferProcess#privateProperties` then, so in subsequent PRs we can reintroduce `TransferProcess#properties` that will be the one from `DataRequest#properties`

Reply:
ok, fine for me